service/tasks/driver.py

```
@@ -1342,14 +1366,12 @@ def add_floating_ip(driverCls, provider, identity, core_identity_uuid,
         # because its a user-supplied problem.
         # Here we will attempt to 'fix' requests and put the 'add_floating_ip'
         # task back on the queue after we're done.
-        celery_logger.exception("Neutron did not accept request - %s."
-                                % bad_request.message)
+        celery_logger.exception("Neutron did not accept request - %s." % bad_request.message)
         if 'no fixed ip' in bad_request.message.lower():
             fixed_ip = add_fixed_ip(driverCls, provider, identity,
                                     instance_alias)
             if fixed_ip:
-                celery_logger.debug("Fixed IP %s has been added to Instance %s."
-                                    % (fixed_ip, instance_alias))
+                celery_logger.debug("Fixed IP %s has been added to Instance %s." % (fixed_ip, instance_alias))
```

Line too long?  There seem to be a number of lines in this file that are getting un-PEP8-ized, is that intentional?

---

I guess it is passing your tests, so maybe you are using longer line length or ignoring this error?

---

Yes, Rob makes the CI ignore that test. I cannot pass the test unless I put it into one line. I think I will stay in this way unless our CI has changed? Thanks for pointing it out.

---

Sure, I just saw a lot of lines that looked like reverse PEP8, wanted to make sure that was on purpose.

---

Thanks!